content/general/channel_lifecycle.textile

```diff
@@ -0,0 +1,124 @@
+- channel.closed := indicates that the channel has been deactivated globally
+- channel.region.active := indicates that the channel has been activated in a specific region
+- channel.region.inactive := indicates that the channel has been deactivated in a specific region. The included @ChannelDetails.status@ includes "occupancy":#occupancy in that region
+- channel.occupancy := indicates a change in global "occupancy":#occupancy. Not all "occupancy":#occupancy changes are sent; there is a minimum interval on a per-channel basis, so if the "occupancy":#occupancy is changing continuously, then only periodic snapshots of occupancy are sent. Further roll-up of events may occur depending on the capacity of the lifecycle channel. Any "occupancy":#occupancy changes in which any of the "occupancy":#occupancy categories changes from 0 to a non-zero count, or vice versa, are always included
+
+h2(#configuration). Configuration
+
+Metachannels have defaults for various attributes - such as the maximum message rate - but certain attributes are also configurable on a per-application basis. These custom configurations are presently only available to "enterprise":https://www.ably.io/pricing/enterprise accounts, and we do not provide dashboard support for modifying that configuration; "contact us":https://www.ably.io/contact to discuss your requirements if this is relevant to you.
+
+The configuration options for the @channel.lifecycle@ metachannel specifically are:
+
+- enabled := _true_ determines whether or not the metachannel is enabled<br>__Type: @boolean@__
+- sendOccupancy := _false_ determines whether or not "occupancy":#occupancy events are enabled<br>__Type: @boolean@__
+- occupancyUpdateInterval := _5000_ where "occupancy":#occupancy events are enabled, the minimum interval, in milliseconds, between "occupancy":#occupancy updates on any given channel<br>__Type: @int@__
+
+h2(#realtime). Subscribing via a realtime connection
+
+The rate of messages published in a metachannel can exceed the rate that a realtime connection can handle. At present, realtime connections are permitted to subscribe to a metachannel but this may result in the connection being frequently detached as the outbound rate limit is hit on the connection. As the rate on a metachannel grows it eventually will scale by having its traffic sharded across multiple independent shard channels, each carrying a fraction of the traffic. Once a metachannel scales to a level that it is sharded, it is no longer possible to subscribe with a realtime connection; it is only possible to get messages from that metachannel via reactor rules.
+
+h2(#metadata-rest). Channel metadata REST API
+
+<%= partial 'shared/_channel_metadata' %>
+
+h2(#enumeration-rest). Channel enumeration REST API
+
+<%= partial 'shared/_channel_enumeration' %>
+
+h2(#reactor). Reactor rules
```

Comment:
This shoudl be grouped with Subscribing.  Logically, we have two sets of APIs.  Realtie APIs and REST APIs.  Let's keep the logic together. Then for Realtime APIs we have events generated when channe lifecycle vents are generated, or if occupancy events are activated, we generate events periodically for as long as channels remain active.